src/transformers/generation/logits_process.py

```diff
@@ -1517,29 +1518,33 @@ class WhisperTimeStampLogitsProcessor(LogitsProcessor):
     ```
     """
 
-    def __init__(self, generate_config):  # support for the kwargs
+    def __init__(
+        self, generate_config, detect_timestamp_from_logprob: Optional[bool] = None
```

Comment:
nit: For consistency, I'd write `detect_timestamp_from_logprob` into `generate_config` in Whisper's `generate`